zk/model/config.py

```diff
@@ -0,0 +1,37 @@
+""" Configuration key store, elements such as event_city or contact_email """
+import sqlalchemy as sa
+from sqlalchemy import orm
+from sqlalchemy.dialects.postgresql import JSON # Required - unsure why
+from meta import Base, Session, metadata
+
+import logging
+log = logging.getLogger(__name__)
+
+class Config(Base):
+    __tablename__ = 'config'
+    category = sa.Column(sa.types.String, primary_key=True)
+    key      = sa.Column(sa.types.String, primary_key=True)
+    value    = sa.Column(sa.dialects.postgresql.JSON)
+    # NOTE: Non-postgresql requires the use of a decorator
+
+    @classmethod
+    def get(cls, category, key=None):
+        """ Get an entry from the config key store
+
+        Calling form:
+        get(category, key) - Fetches the specified value from the database
+        get(key) - Fetches the value from the database with the general category
+        """
+
+        if key is None:
+            key = category
+            category = 'general'
```

Comment:
My preference would be to use category as a keyword. For example:

def get(key, category='general'):

Then you call:
get(key)
get(key, 'altcat')
Or even neater:
get(key, category='altcat')